### ISSUE TITLE
Help text additions to Molarity PDOM

### ISSUE THREAD

**Issue by emily-phet** [phetsims/molarity#83]
@twant 

@Matthew-phet @terracoda and I discussed the recent interviews on Molarity, and felt that the addition of the following help text would likely be helpful for users.

- [x] Addition of help text just above keyboard shortcuts hint in scene summary (see A11y main design for specific text for these).

- [x] Addition of help text just 1) below the solute amount slider, 2) below solution volume slider, and 3) above combobox. Please see A11y design main for specific text for these.

These help text have no changing parameters, and remain static throughout sim use. Please let me know if you have any questions.

**Comment by twant** [comment]
@emily-phet and @Matthew-phet I'll take a look and get you a dev version once its been implemented. Question: from looking at the design doc, it looks like this text should be a part of the PDOM. Is this correct, or do you want it to be read out on focus of those elements?

**Comment by terracoda** [comment]
@twant, yes that is correct. The help text is not read out automatically, just discoverable by the user when they read it in the PDOM's main content.

**Comment by twant** [comment]
@zepumph and I renamed the option buttonLabelContent to buttonAccessibleName for API clarity while working on adding help text for the solute combo box.

**Comment by twant** [comment]
Updated, I believe! Check it out here and see what you think, @Matthew-phet @emily-phet 

https://phet-dev.colorado.edu/html/molarity/1.5.0-dev.10/phet/molarity_en_phet.html

**Comment by twant** [comment]
@emily-phet @Matthew-phet checking on this issue to see if you think it's been implemented correctly and can be closed. Thanks!
